src/Symfony/Component/Console/Helper/TreeStyle.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Helper;
+
+/**
+ * Configures the output of the Tree helper.
+ *
+ * @author Simon André <smn.andre@gmail.com>
+ */
+final class TreeStyle
+{
+    public function __construct(
+        private readonly string $prefixEndHasNext,
+        private readonly string $prefixEndLast,
+        private readonly string $prefixLeft,
+        private readonly string $prefixMidHasNext,
+        private readonly string $prefixMidLast,
+        private readonly string $prefixRight,
+    ) {
+    }
+
+    public static function box(): self
+    {
+        return new self(
```

Comment:
on one line things would be better visually - not consuming my screen's space for nothing ;)